data/scenarios/Testing/harvest.yaml

```diff
@@ -0,0 +1,36 @@
+name: Test harvest command
+description: Test the difference between the grab and harvest commands.
```

Comment:
```suggestion
description: |
  Test the difference between the grab and harvest commands.
  https://github.com/swarm-game/swarm/issues/490
```


I admit I like being able to click on the PR link in the terminal:
![Screenshot from 2022-06-29 23-22-28](https://user-images.githubusercontent.com/44544735/176547532-f0a747d2-7a27-462f-adb5-24409ca4bafa.png)